File: ruby/arrays/fizzbuzz.rb
def fizzbuzz(arr)
  fizzbuzz_arr = []
  
  arr.each do |number|
    if number % 3 == 0 && number % 5 == 0 
      number = "fizzbuzz"
      fizzbuzz_arr << number
    elsif number % 5 == 0 
      number = "buzz"
      fizzbuzz_arr << number
    elsif number % 3 == 0 
      number = "fizz" 
      fizzbuzz_arr << number
    else 
      fizzbuzz_arr << number 
    end 
  end
  
  return fizzbuzz_arr.join(" ")
  
end

File: javascript/arrays/fizzbuzz.js
function fizzbuzz(arr) {
  for (i=0; i < arr.length; i++) {
    if (arr[i] % 3 === 0 && arr[i] % 5 === 0) {
      console.log("fizzbuzz")
    }
    else if (arr[i] % 5 === 0) {
      console.log("buzz")
    } 
    else if (arr[i] % 3 === 0) {
      console.log("fizz")
    } else {
      console.log(arr[i])
    }
  }
}

var arr = Array.from(Array(100).keys())

return fizzbuzz(arr)